### PR TITLE
fix: #62 (Login from Navbar not working)

### PR DESCRIPTION
**Pull request by kartikpandey2**
Explain the **details** for making this change. What existing problem does the pull request solve?
[Issue](https://github.com/scorelab/fact-Bounty/issues/62)

### CODE REVIEW

File: fact-bounty-client/src/shared/components/TopNavBar.js [hunk 1]
```diff
@@ -45,8 +45,7 @@ class TopNavBar extends Component {
   	this.setState(state => ({
     	anchorEl: null
   	}))
-  }
-
```
Comment (Anmolbansal1) [inline fact-bounty-client/src/shared/components/TopNavBar.js]: I think this is an unnecessary change

**Comment by ivantha** [comment]
@root5533 Please review this.

**Comment by root5533** [comment]
Rebase with master branch and squash commits

**Comment by kartikpandey2** [comment]
@root5533  Is that what you asked or I have done something wrong..? 😅

**Comment by root5533** [comment]
You haven't rebased it properly @kartikpandey2. There are duplicate commits in the PR. Please look into your branch and sync with master branch.

**Comment by kartikpandey2** [comment]
@root5533  Is it fine now.? I removed duplicate commit 😅

**Comment by kartikpandey2** [comment]
@Anmolbansal Now this fine .?
